micropip/_compat/compatibility_layer.py

```diff
@@ -13,14 +13,6 @@ class CompatibilityLayer(ABC):
     All of the following methods / properties must be implemented for use both inside and outside of pyodide.
     """
 
-    class HttpStatusError(ABC, Exception):
```

Comment:
Please remove it from [here](https://github.com/pyodide/micropip/blob/71bc2e3a5eec9c842f466dd0b5944631feb3b96c/micropip/_compat/_compat_in_pyodide.py#L10) and [here](https://github.com/pyodide/micropip/blob/71bc2e3a5eec9c842f466dd0b5944631feb3b96c/micropip/_compat/_compat_not_in_pyodide.py#L20-L27) too.

Reply:
Done in abc1437ae6bc3e67718e51c3b91ae7910f4124f6; I didn't realise you were asking to remove `HttpStatusError`s completely.